### DISCUSSION TITLE
Have some doubts

### DISCUSSION THREAD

**Discussion by CasperClous** [postfixadmin/postfixadmin#460] [Q&A]
Hello, I'm a student making research, and I have some problems, I want to know (I have been searching), how the msg-id is been made. If you know plz let me know, where is the file that makes the msg-id.

**Comment by DavidGoodwin** [comment]
This sounds like a Postfix question; not a PostfixAdmin question.

Try reading http://www.postfix.org/postconf.5.html - look for enable_long_queue_ids - which mentions some bits about it.
